check.py

```diff
@@ -13,7 +20,9 @@ def _count_lines(path, chunk_size=8 * 10**6):
     return count
 
 
-def get_info_about_all_loaded_lines(run_accession, path="."):
+def get_info_about_all_loaded_lines(
+    run_accession: str, path: Union[str, PathLike] = "."
```

Comment:
Could you please name a case where this `path` argument would be of a `str` type and not exactly `pathlib.Path`? 

From this line
```accession_directory = Path(output_directory, accession)```
it seems as if it is always `Path`

Reply:
Type hints in a function signature aren't about the arguments that we pass in now, but rather about the arguments the function can handle. `get_info_about_all_loaded_lines` can accept `str` and `Path`-like objects, is there a good reason why we should restrict it to `Path` only?

Reply:
Good point. In this case I suggest we make an alias for this path type, something like
```python
PathType = Union[str, PathLike]
```
put it to a new file called `typing_helpers.py` and import from it